.eslintrc.json

```diff
@@ -1,9 +1,14 @@
 {
     "env": {
         "browser": true,
-        "commonjs": true
+        "commonjs": true,
+        "es6": true,
+        "node": true
```

Comment:
The es6 is to allow module exporting in the future, the node is because the code can be in a server side rendering environment and the commonjs is for the module.exports statement I think.